demo/app/script.ts

```diff
@@ -26,10 +26,18 @@ space.on('membersUpdate', (members) => {
 });
 
 /** Avoids issues with top-level await: an alternative fix is to change build target to es */
-(async () => {
+
+const init = async () => {
   await space.enter({ name: selfName });
-  renderSelfAvatar(selfName);
-  renderFeatureDisplay(space);
-  const initialMembers = space.getMembers();
-  renderAvatars(initialMembers.filter(memberIsNotSelf));
+};
+
+(async () => {
+  space.once('membersUpdate', () => {
```

Comment:
Why can't we just use space.locations.set(`slides-2`) in the async init operation after `space.enter` instead of running this extra update?
How does this `space.once` call solve the issue from the ticket?